tools/package_gcsfuse_docker/Dockerfile

```diff
@@ -52,7 +52,8 @@ RUN fpm \
     -d fuse \
     --vendor "" \
     --url "https://$GCSFUSE_REPO" \
-    --description "A user-space file system for Google Cloud Storage."
+    --description "A user-space file system for Google Cloud Storage." \
+    --after-install "${GCSFUSE_PATH}/tools/logrotate_config/package_after_install_script.sh"
```

Comment:
As discussed, confirm if this installation is done at the time package is installed on a machine and what happens if the machine/docker container on which this package is installed doesn't have logrotate library installed.

Reply:
I checked and found gcsfuse gets installed irrespective of what verdict we get as an execution of post-script. Hence, started applying all these configuration as before-script so in case of failure we will abort the gcsfuse installation.